README.md

```diff
@@ -1,2 +1,4 @@
 # helloworld
 1st
+Was a computer prodigy at 12.  Emplyed by large security firm (yes its true)  I suffered 15 years of heroin addiction and no jobs are available to me.  This is my passion, the how I make it through the day drug free.  Hopefully a career as ex cons arn't looked at with the irhgtest of lights!  PLEASE, I beg of you humbly and sincerely, help me to learn what I once knew.  The learning curve over almost 25 years is astrnomical - exponentially so.  But I have the right brain and intuition, and the motivation for this to become a reality.
```

Comment:
+Was a computer prodigy at 12.  Emplyed by large security firm (yes its true)  I suffered 15 years of heroin addiction and no jobs are available to me.  This is my passion, the how I make it through the day drug free.  Hopefully a career as ex cons arn't looked at with the irhgtest of lights!  PLEASE, I beg of you humbly and sincerely, help me to learn what I once knew.  The learning curve over almost 25 years is astrnomical - exponentially so.  But I have the right brain and intuition, and the motivation for this to become a reality.